src/auth.md

```diff
@@ -143,6 +143,7 @@ This example demonstrates how to integrate a browser based application with the
 Centrapay OAuth server using the [oidc-client-js][oidcjs]{:.external}
 JavaScript library.
 
+<span id="oidcjs-example">
```

Comment:
Should this span be closed?

Reply:
Yes

Reply:
Looks like this was added in error. There were two IDs on the page which wasn't so great.

I've removed the span so we should be golden 🕺